librz/core/project.c

```diff
@@ -44,19 +44,20 @@ RZ_API RzProjectErr rz_project_save_file(RzCore *core, const char *file, bool co
 
 	if (compress) {
 		int mkstemp_fd = rz_file_mkstemp("svprj", &tmp_file);
-		close(mkstemp_fd);
```

Comment:
why this?

Reply:
In case of `mkstem_fd` is -1 `close()` is unadvised AFAIK.